android/build.gradle

```diff
@@ -19,7 +19,7 @@ buildscript {
     dependencies {
         // Matches recent template from React Native (0.59)
         // https://github.com/facebook/react-native/blob/0.59-stable/template/android/build.gradle#L16
-        classpath 'com.android.tools.build:gradle:4.1.0'
+        classpath 'com.android.tools.build:gradle:7.1.3'
```

Comment:
7.4.2 is the latest release. https://gradle.org/install/

Reply:
This is the the android gradle plugin, not gradle itself.

Reply:
Ahhh, thanks for the clarification.